back/src/Domain/Buying/Buying.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Buying;
+
+use App\Domain\Common\Behavior\TimestampableTrait;
+use App\Domain\Common\Behavior\UlidTrait;
+use App\Domain\Apiary\Apiary;
+use Symfony\Component\DependencyInjection\Attribute\Exclude;
+
+#[Exclude]
+class Buying
+{
+    use UlidTrait;
+    use TimestampableTrait;
+
+    private string $label;
+    private float $price;
+    private Apiary $apiary;
+
```

Comment:
On ajoutera le champs User afin de moins galérer avec l'auth côté API (en gros, on s'évite de devoir passer par Apiary pour vérifier si l'entité correspond bien à l'utilisateur courant. D'autant plus si maintenant $apiary est nullable.)
```suggestion
    private User $user;
```